lib/db/ignored_files_db.dart

```diff
@@ -106,6 +106,35 @@ class IgnoredFilesDB {
     return result;
   }
 
+  Future<void> removeIgnoredEntries(List<IgnoredFile> ignoredFiles) async {
+    final startTime = DateTime.now();
+    final db = await instance.database;
+    var batch = db.batch();
+    int batchCounter = 0;
+    for (IgnoredFile file in ignoredFiles) {
+      if (batchCounter == 400) {
```

Comment:
Why do we need to batch them?

Reply:
Just followed the current practice for doing batch operation when there are multiple entries which need insertion/updation etc.